membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -74,15 +74,24 @@ class AttributeController extends Controller with LoggingWithLogstashFields {
         case Some(identityId) =>
           val (fromWhere, attributes) = pickAttributes(identityId)
           attributes.map {
-            case Some(attrs @ Attributes(_, Some(tier), _, _, _, _, _)) =>
-              log.info(s"$identityId is a member - $endpointDescription - $attrs found via $fromWhere")
+            case Some(attrs @ Attributes(_, Some(tier), _, _, _, _, _, _)) =>
+              log.info(s"$identityId is a $tier member - $endpointDescription - $attrs found via $fromWhere")
               onSuccessMember(attrs).withHeaders(
                 "X-Gu-Membership-Tier" -> tier,
                 "X-Gu-Membership-Is-Paid-Tier" -> attrs.isPaidTier.toString
               )
+            case Some(attrs @ Attributes(_, _, _, _, _, _, _, Some(date))) =>
+              log.info(s"$identityId is a digital subscriber expiring $date - $endpointDescription - found via $fromWhere")
```

Comment:
Might it be worth logging $attrs in these cases still since someone could match more than one case?

Reply:
I'd meant to make this only digital subscriber, so I'll fix that and let the user with multiple products scenario fall down to the last case.